.github/workflows/sonarcloud.yml

```diff
@@ -86,9 +86,7 @@ jobs:
           timeout_minutes: 30
           max_attempts: 2
           retry_wait_seconds: 30
-          command: |
-            cd prime-router
-            ./gradlew -Dorg.gradle.jvmargs="-Xmx6g" package -x fatjar
+          command: ./gradlew -Dorg.gradle.jvmargs="-Xmx6g" :prime-router:package -x fatjar
```

Comment:
outside the scope of this ticket but are we really using 6 gigs of heap space?

Reply:
ah wait, this command itself just requires 6 gigs (which also seems incredibly high but not a big deal as a one-off during ci)

Reply:
The sonarcloud action does require 6GB